cloudify/client.go

```diff
@@ -23,26 +23,55 @@ import (
 	"io/ioutil"
 )
 
+// ClientConfig - all configuration fields for connection
+type ClientConfig struct {
+	Host      string
+	User      string
+	Password  string
+	Tenant    string
+	Debug     bool
+	AgentFile string
+}
+
 //Client - struct with connection settings for connect to manager
 type Client struct {
-	restCl rest.ConnectionOperationsInterface
+	ClientConfig
```

Comment:
@0lvin Do not you think that ClientConfig should be something like this 
```
ClientConfigObj  ClientConfig
```
I mean thats the way how we define variable inside struct on go

Reply:
I like to inherit from ClientConfig(not include as field).